granulate_utils/linux/cgroups/base_cgroup.py

```diff
@@ -55,6 +56,10 @@ def cgroup_mount_path(self) -> Path:
     def cgroup_path(self) -> Path:
         return Path(self.get_cgroup_hierarchies()[self.subsystem])
 
+    @property
```

Comment:
This is a heavy operation and is not cached - should be a function IMO

Reply:
Indeed, from PEP8:
> Avoid using properties for computationally expensive operations; the attribute notation makes the caller believe that access is (relatively) cheap.

Fixed.